src/geometry.cpp

```diff
@@ -60,3 +65,142 @@ bool Geometry::reloadFile(size_t maxVertexCount)
     return loadFile(m_fileName, maxVertexCount);
 }
 
+void Geometry::initializeGL()
+{
+    this->destroyBuffers();
+
+    this->initializeBboxGL(this->shaderId("boundingbox"));
+}
+
+void Geometry::destroyBuffers()
+{
+    // destroy any previously created buffers (in case we are re-initializing this geometry)
+    // this should avoid recreating more and more opengl buffers
+
+    //tfm::printfln("Geometry :: destroyBuffers");
+
+    for (std::map<std::string,unsigned int>::iterator it=m_VBO.begin(); it!=m_VBO.end(); ++it)
```

Comment:
I think at this stage all the compilers we care about have supported `auto` for some time, so `it` can be declared `auto`.